python/ray/tune/tuner.py

```diff
@@ -167,6 +167,7 @@ def restore(
         resume_unfinished: bool = True,
         resume_errored: bool = False,
         restart_errored: bool = False,
+        with_parameters: Optional[Dict[str, Any]] = None,
```

Comment:
is there a way to mark if the original trainable was a `with_parameters` wrapped one, and then raise an error based on that information if this arg is passed in?

Reply:
nvm ignore this- this is already being done, I just missed this!